### DISCUSSION TITLE
Should I remove the Snap version?

### DISCUSSION THREAD

**Discussion by cboxdoerfer** [cboxdoerfer/fsearch#364] [Polls]
I've noticed a couple of issues with the Snap version in particular, which make me wonder if I should remove it from the Snapcraft store.

https://snapcraft.io/fsearch

Due to the sandboxing a Snap application can't access certain files/folders on the system. For example by default you won't be able to search in removable storage devices. It's also impossible to access many hidden files/folders. I've made sure to mention those issues in the description of the application in the store (including a recommendation to use the PPA instead), but users seem to either ignore, don't understand or don't read it, and then either stop using the application or leave a bad review, like:

> (1/5) can't access external drives: what's the point if it can't even access external drives?

At that point the next and IMHO worst issues arises: The Ubuntu/GNOME Software Center doesn't support responses to reviews, which means FSearch now has bad reviews and I can't even explain what went wrong and that it's not the application to blame. I also don't get any debug information, for example the software center doesn't tell me which FSearch version the reviewer was using. So the experience from a developer's point of view is quite bad and to me it just looks like bad press and is quite demotivating.

So I'm wondering what you think? Like are some of you using the Snap version and rely on it? According to the Snapcraft stats there are roughly 750 unique snap users per week using FSearch, so the user base isn't that big, but not too small either.

Kind regards,
Christian

**Comment by cboxdoerfer** [comment]
After discovering more issues with the Snap package and after the request to have FSearch under classic confinement (no sandbox) was declined by Snap developers, I've decided to no longer support Snap. More details can be found here: https://github.com/cboxdoerfer/fsearch/wiki/Snap-is-no-longer-officially-supported

**Reply by AtomicRobotMan0101** [reply]
I get the reason for snaps existence, but the absolute adherence to it for *everything* is insane.

Firejail... perhaps... Flatpak... mayyyybe... but this whole "virtualise and wrap Just Because" is nuts.

What is REALLY needed is a bloody good clean up of the FHS into something that makes SENSE. Its a mess.
